scripts/zones/Southern_San_dOria/npcs/Diary.lua

```diff
@@ -15,7 +15,7 @@ end
 entity.onTrigger = function(player, npc)
     local aSquiresTestII = player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.A_SQUIRE_S_TEST_II)
     local medicineWoman = player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.THE_MEDICINE_WOMAN)
-    local toCureaCough = player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.TO_CURE_A_COUGH)
+    local toCureaCough = player:getCharVar("toCureaCough");
```

Comment:
```suggestion
    local toCureaCough = player:getCharVar("toCureaCough")
```

Reply:
Whoops. No semi-colons in lua, got it. That'll be a hard habit to break. Thanks ;)

Reply:
it totally is lol. easy fix!